otterdog/eclipse-tycho.jsonnet

```diff
@@ -66,6 +69,26 @@ orgs.newOrg('eclipse-tycho') {
           value: "********",
         },
       ],
+      rulesets: [
+        orgs.newRepoRuleset('main') {
+          allows_updates: true,
+          bypass_actors+: [
+            "@eclipse-tycho/technology-tycho-committers"
+          ],
+          include_refs+: [
+            "refs/heads/main",
+          ],
+          required_approving_review_count: 0,
+          required_status_checks+: [
+            "continuous-integration/jenkins/pr-head",
+            "eclipsefdn/eca",
+            "License vetting status check"
```

Comment:
I'm not totally sure about this, what I found in some examples it seem to need match a name.

Basically I want that this check is mandatory:
https://github.com/eclipse-tycho/tycho/actions/workflows/licensecheck.yml

Reply:
the additional eclipsefdn/eca check is is not needed, as it is already inherited from the default.

For the license checking status, you should use

"call-license-check / check-licenses"

I checked it via the web ui and that the status that is detected. It follows the rules that are explained here: https://otterdog.readthedocs.io/en/latest/reference/organization/repository/status-check/

the jenkins status is correct afacit.

Reply:
here is a screenshot from the web ui:

![image](https://github.com/eclipse-tycho/.eclipsefdn/assets/1048055/523124e4-448a-4823-83da-ff4b61e41f83)

Reply:
Is there a link to the webui? Is it possible to edit the project file with that so one don't need to edit file by hand?